tests/integrationtest/t/infoschema/infoschema.test

```diff
@@ -58,3 +58,14 @@ select engine, DATA_LENGTH from information_schema.tables where (table_name ='t4
 explain select engine, DATA_LENGTH from information_schema.tables where table_name ='t4' and upper(table_name) ='T4' and table_schema = 'infoschema__infoschema';
 select engine, DATA_LENGTH from information_schema.tables where table_name ='t4' and upper(table_name) ='T4' and table_schema = 'infoschema__infoschema';
 
+# TestPartitionsColumn
+create table pt1(a int primary key, b int) partition by hash(a) partitions 4;
+create table pt2(a int primary key, b int) partition by hash(a) partitions 4;
+-- sorted_result
```

Comment:
I'm not familiar with these logic. Can these test make sure the execution does not query full data in TiKV?

Reply:
No. It only make sure the result is correct. Let me add a manual test about `information_schema.partitions`.